src/utils/formatting.ts

```diff
@@ -125,14 +123,21 @@ export const formatLiveEventMetaString = (media: PlaylistItem) => {
   return metaData.join(' • ');
 };
 
-export const formatVideoSchedule = (scheduledStart?: Date, scheduledEnd?: Date) => {
+export const formatVideoSchedule = (locale: string, scheduledStart?: Date, scheduledEnd?: Date) => {
   if (!scheduledStart) {
     return '';
   }
 
   if (!scheduledEnd) {
-    return format(scheduledStart, 'PPP • p');
+    return formatLocalizedDateTime(scheduledStart, locale, '•');
   }
 
-  return `${format(scheduledStart, 'PPP • p')} - ${format(scheduledEnd, 'p')}`;
+  return `${formatLocalizedDateTime(scheduledStart, locale, '•')} - ${formatLocalizedTime(scheduledEnd, 'locale')}`;
 };
+
+const formatLocalizedDate = (date: Date, locale: string) => new Intl.DateTimeFormat(locale, { day: 'numeric', month: 'long', year: 'numeric' }).format(date);
+
+const formatLocalizedTime = (date: Date, locale: string) => new Intl.DateTimeFormat(locale, { hour: 'numeric', minute: 'numeric' }).format(date);
+
+const formatLocalizedDateTime = (date: Date, locale: string, separator = ' ') =>
+  `${formatLocalizedDate(date, locale)} ${separator} ${formatLocalizedTime(date, locale)}`;
```

Comment:
```ts
const formatLocalizedDateTime = (date: Date, locale: string, separator = '') =>
  `${formatLocalizedDate(date, locale)} ${separator} ${formatLocalizedTime(date, locale)}`;
```
or

```ts
const formatLocalizedDateTime = (date: Date, locale: string, separator = ' ') =>
  `${formatLocalizedDate(date, locale)}${separator}${formatLocalizedTime(date, locale)}`;
```

Now, without a separator, it render three whitespaces between the date and time:

`30-05-2023   09:09`

I think the last example is the most flexible because you can either render a single whitespace or a character padded with whitespaces. If you don't like the separator being a whitespace or a whitespace padded character, we could also do something like this:

```ts
const formatLocalizedDateTime = (date: Date, locale: string, separator = '') =>
  [formatLocalizedDate(date, locale), separator, formatLocalizedTime(date, locale)].filter(Boolean).join();

formatLocalizedDateTime(new Date(), 'nl', '');  // 30-05-2023 09:09
formatLocalizedDateTime(new Date(), 'nl', '•'); // 30-05-2023 • 09:09
```